### PR TITLE
Timer polish / Admin tools

### PR DESCRIPTION
**Pull request by sparrowDom**
- minor fixes so that claim page re-renders when timer reaches 0
- homepage redirects to claims by default (we should change that when we enable proposal creation)
- Adds Admin tools to be able to override claim open/closes timers (see screenshot below). Menu is enabled by running `localStorage.setItem("admin", "true");` from the browser console

Screenshot 2022-07-08 at 14 02 54
 
**fixes:** https://github.com/OriginProtocol/ousd-governance/issues/171 https://github.com/OriginProtocol/ousd-governance/issues/184

### CODE REVIEW

File: client/components/holding/Page.tsx [hunk 1]
```diff
@@ -29,86 +30,89 @@ const renderer: CountdownRendererFn = ({ days, hours, minutes, seconds }) => (
   </CardGroup>
 );
 
-let date = new Date(0);
-date.setUTCSeconds(parseInt(process.env.CLAIM_OPENS || "0"));
+const HoldingPage = () => {
+  const { claim } = useStore();
+  let date = new Date(0);
+  date.setUTCSeconds(parseInt(claim.claimOpensTs || "0"));
```
Comment (sparrowDom) [inline client/components/holding/Page.tsx:37]: haven't changed anything in this file below this comment...

File: client/utils/index.tsx [hunk 2]
```diff
@@ -123,23 +123,23 @@ const fetcher = (url: string) => fetch(url).then((res) => res.json());
...
+  return parseInt(useStore.getState().claim.claimClosesTs || "0");
 }
 
-const now = Math.floor(new Date().getTime() / 1000);
+const now = () => Math.floor(new Date().getTime() / 1000);
 
 export function claimOpenTimestampPassed() {
   if (!claimOpensTimestamp()) return true;
-  return now > claimOpensTimestamp();
+  return now() > claimOpensTimestamp();
```
Comment (sparrowDom) [inline client/utils/index.tsx:137]: This was needed otherwise a "cached" now time was used.

**Review by tomhirst — COMMENTED** [review]
This is super cool.

LGTM except for 1 small nitpick. We get a console error on the initial load. It looks like the server-side content is mismatching with the client-side, which Next.js doesn't like. I think it's the timer seconds:

Screenshot 2022-07-08 at 13 35 04

File: client/utils/index.tsx (hunk 2 above)
Comment (tomhirst) [reply client/utils/index.tsx:137]: Smart.

**Comment by sparrowDom** [comment]
tnx @tomhirst... fixed with useEffect and timeout, instead of passing date object from the page down to the component (if server side props would be used)... annoying code messy workaround -.-

**Pull request merged**